go/runtime/bundle/manager.go

```diff
@@ -437,6 +472,65 @@ func (m *Manager) fetchBundle(url string) (string, error) {
 	return file.Name(), nil
 }
 
+func (m *Manager) cleanStaleBundles(runtimeID common.Namespace) {
```

Comment:
```suggestion
func (m *Manager) cleanBundles(runtimeID common.Namespace) {
```
I would keep this simple.

Reply:
we also have `cleanupBundles`. maybe we rename it `cleanupBundlesInit` or `cleanBundlesInit` as it is only called during initialization of worker?

Reply:
Name `cleanupBundlesInit` is probably not the best, as this is not `Init` method. Maybe `cleanupBundlesOnStartup`, or something shorter like `clearBundles`,  `discardBundles`, `pruneBundles`.